restock-shop.lic

```diff
@@ -224,50 +201,82 @@ class ShopRestock
       end
     end
 
-    while bput('get silk cloth from my back', 'You get', 'What were') == 'You get'
+    while bput("get silk cloth from my #{@bag}", 'You get', 'What were') == 'You get'
       fput('combine cloth')
     end
     stow_hands
 
     missing_items.each do |item|
       move('out') if Room.current.id == @inner_id
-      wait_for_script_to_complete('sew', ['stow', 'sewing', item[4], item[5], item[0]])
-      walk_to(@shop_id)
-      fput('open entrance')
-      pause 0.5
-      fput('go entrance')
-      fput("get #{item[0]} from back")
-      fput('get my stamp')
-      fput("mark my #{item[0]} with my stamp")
-      waitrt?
-      pause
-      fput('stow stamp')
-      fput("sell my #{item.first} #{item[3]} for #{item[1]}")
+      wait_for_script_to_complete('sew', ['stow', 'sewing', item['chapter'], item['recipe'], item['noun']])
```

Comment:
❓ Does this need 'silk' as a new argument at the end? I'm looking at how lines 134, 175, and 234 work; and this is different.

Reply:
I believe so, thanks for catching that

Reply:
👍